packages/lib-utils/src/types/redux.ts

```diff
@@ -1,7 +1,7 @@
 import type { Map as ImmutableMap } from 'immutable';
 import type { K8sResourceCommon } from './k8s';
 
-export type K8sState = ImmutableMap<string, unknown>;
+export type K8sState = ImmutableMap<string, unknown> | undefined;
```

Comment:
Reducer state can never been undefined save from the first invocation... I would not type this with `| undefined`... what's the use-case you're trying to solve?

Reply:
Maybe we leave this too -- this map needs to be fixed but maybe not for this PR for us to move forward.